util.go

```diff
@@ -179,14 +179,16 @@ func matchRule(history ring, curVal, ruleMin, ruleAbs, ruleDiff, ruleMax int) (b
 	}
 
 	// the current peak load exceed the absolute value
-	if curVal > ruleAbs {
+	if curVal > ruleAbs && ruleAbs!=0{
```

Comment:
it's better to add a new condition at the front of this function, that will make the code simpler.